src/resource/source.rs

```diff
@@ -50,6 +54,9 @@ pub struct Args {
     /// The URL of the data source.
     pub remote: String,
 
+    /// Set to true if you want to avoid date expansion into year, day of week, etc.
+    pub disable_datetime: Option<bool>,
```

Comment:
(You know, I really ought to improve the macro expander for `#[updatable]` to support something similar for `#[create_arg]`, so we can auto-generate the `Arg` structs and not duplicate struct members like we do here. Maybe someday—it's overkill, but it would make the code a bit more regular, not having two different ways to declare related structs.)